### DISCUSSION TITLE
[Suggestion] Recipe Confilct avoidance through random output

### DISCUSSION THREAD

**Discussion by drakray** [vectorwing/FarmersDelight#904] [Ideas]
Hi, could it be possible to make the Cutting Board support conflicting recipe, by selecting one of different available output each cut?
Example of this would be the Potato cutting recipe from Baked Delight, More Delight, and Casualness Delight.
Each addons add a potato cutting recipe, but only one is really available, and Polymorph is not available for the cutting board

**Comment by Aceplante** [comment]
You can fix that with datapacks
